test/default.js

```diff
@@ -70,5 +83,54 @@ metatests.test('must properly process schemas', async test => {
     test.strictSame(category, categories[name]);
   }
 
+  const [addDomains] = config.processors.domains.add;
```

Comment:
This looks like a unit test, you don't need to load schemas from filesystem to check that `add` works the way it is supposed to work.